help/de/Help_ch_Account-Actions.xml

```diff
@@ -89,337 +103,350 @@
+            <entry namest="c4">
+              <para>nur Eigenkapitalkonten
               </para>
             </entry>
+          </row>
 
-            <entry namest="c3" morerows="2">
-              <para>alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten
+          <row rowsep="1">
+            <entry namest="c1" nameend="c2">
+              <para>Erfolgskonten
+                <footnote id="fn-quick-kat">
+                  <para>Wird in <application>Quicken</application> bzw. <application>Lexware FinanzManager</application> und
+                    anderen, nicht auf doppelter Buchführung basierenden, Programmen als
+                    <quote>Kategorie</quote> bezeichnet.
+                  </para>
+                </footnote>
+              </para>
+            </entry>
+
+            <entry namest="c3" nameend="c4">
+              <para>Bei Bedarf können Sie auf der nächsten Kontoebene die Unterscheidung zwischen
+                steuerpflichtig/-frei, bzw. steuerabzugsfähig oder nicht einfügen. Mittels
+                <menuchoice>
+                  <guimenu>Be<accel>r</accel>ichte</guimenu><guisubmenu>Aufwand &amp;
+                  <accel>E</accel>rtrag</guisubmenu><guimenuitem>Gewi<accel>n</accel>n- und
+                  Verlustrechnung</guimenuitem>
+                </menuchoice>
+                können Sie eine Aufstellung erzeugen.
               </para>
             </entry>
           </row>
 
           <row>
             <entry namest="c1">
-              <para>Investmentfonds
+              <para>Erträge
               </para>
             </entry>
 
-            <entry namest="c2">
-              <para>Ein professionell verwaltetes Portfolio aus Aktien, Anleihen oder anderen Geldanlagen, die sich in
-                Anteile teilen lassen.
+            <entry namest="c3">
+              <para>Einnahmen jeder Art, die Sie z.B. durch Gehalt, geldwerte Vorteile (Dienstwagen), Zinsen, Dividenden
+                oder auch Geschenke usw. haben. <footnoteref linkend="fn-quick-kat" />
+<!-- footnoteref funktioniert in Yelp nicht ! -->
+              </para>
+            </entry>
+
+            <entry namest="c4" morerows="1">
+              <para>nur Aufwands- und Ertragskonten
               </para>
             </entry>
           </row>
 
           <row>
             <entry namest="c1">
-              <para>Aktienkonto
+              <para>Aufwand
               </para>
             </entry>
-
-            <entry namest="c2">
-              <para>Aktien oder Anteile an einer Gesellschaft, die den Eigentümer an all ihren Gewinnen und Risiken
-                beteiligen.
+<!-- FIXME source: insert footnote -->
+            <entry namest="c3">
+              <para>Ausgaben jeder Art, z.B. Steuern und Abgaben, Miete, Lebensmittel, Kleidung uvm.
+                <footnoteref linkend="fn-quick-kat" />
               </para>
             </entry>
```

Comment:
```

          <row rowsep="1">
            <entry namest="c1" nameend="c2">
              <para>Interne Konten
              </para>
            </entry>

            <entry namest="c3" nameend="c4">
              <para>Interne Konten werden nicht vom Benutzer, sondern bei Bedarf vom Programm angelegt.
              </para>
            </entry>
          </row>

          <row rowsep="1">
            <entry>
              <para>Status
              </para>
            </entry>

            <entry>
              <para>Kontoart/-name
              </para>
            </entry>

            <entry>
              <para>Beschreibung
              </para>
            </entry>

            <entry>
              <para>übergeordnetes Konto (Hauptkonto)
              </para>
            </entry>
          </row>

          <row rowsep="1">
            <entry namest="c1">
              <para>OK
              </para>
            </entry>

            <entry namest="c2">
              <para>Devisenhandel
              </para>
            </entry>

            <entry namest="c3">
              <para>Devisenhandelskonten repräsentieren/gleichen aus das schwebende Ergebnis Ihres Wertpapier- und
                Devisenhandels. Sobald sie den ersten Kauf tätigen, etwa Umtausch Ihrer
                Heimwährung in die des Urlauslandes, wird für beide Währungen ein Konto angelegt,
                sofern es nicht bereits existiert. Die Details sind in
                <ulink url="&tit-sel-tut;"></ulink> und <ulink url="&tit-sel-gnc;"></ulink> in
                Englisch beschrieben.
              </para>
            </entry>

            <entry namest="c4">
              <para>nur Devisenhandelskonten
              </para>
            </entry>
          </row>

          <row rowsep="1">
            <entry namest="c1">
              <para>Fehler
              </para>
            </entry>

            <entry namest="c2">
              <para>Ausbuchungskonto-<replaceable>XXX</replaceable>, Ausgleichskonto-<replaceable>XXX</replaceable>
              </para>
            </entry>

            <entry namest="c3">
              <para>Sie haben mindestens eine unausgeglichene Buchung eingegeben. [wann welcher Name?] Zur Behebung
                verschieben sie die enthaltenen Buchungen in die richtigen Konten. Anschließend
                können sie das Konto löschen.
              </para>
            </entry>

            <entry namest="c4">
              <para>nur Aktiva [?]
              </para>
            </entry>
          </row>

```
Bitte überprüfen. In dieselbe Tabelle oder neue, –

Reply:
habe ich direkt unten angehängt. Wenn ich das in GC richtig sehe, kann das übergeordnete Konto des Ausgleichskontos sowohl Aktiva als auch Passiva sein.